docs/FAQ/General.md

```diff
@@ -158,6 +158,22 @@ Be sure to enable the webhook and hit the `Add webhook` to save it.
 
 ![Create new Webhook](../img/FAQ/btcpayWebhookFAQ2.jpg)
 
+## Webhook format is not bitpay compatible?
+
+Webhooks are not meant to be compatible with bitpay API. 
+There's two seperate IPNs in BTCPay Server.
+
+* Webhooks 
+* notifications (In BitPay terms: "Instant Payment Notifications")
+
+Where Webhooks are the GreenField Events and Notifications are Bitpay events.
```

Comment:
Nitpick: never saw Greenfield API to be camel cased like here so maybe better to keep it like we have it also on the [API docs](https://docs.btcpayserver.org/API/Greenfield/v1/)?

Reply:
ack